src/Symfony/Component/Serializer/Tests/DependencyInjection/SerializerPassTest.php

```diff
@@ -83,17 +83,20 @@ public function testServicesAreOrderedAccordingToPriority()
 
     public function testBindSerializerDefaultContext()
     {
+        $context = ['enable_max_depth' => true];
+
         $container = new ContainerBuilder();
         $container->setParameter('kernel.debug', false);
-        $container->register('serializer')->setArguments([null, null]);
```

Comment:
These changes shouldn't be necessary.

Reply:
Because of `replaceArgument`, I think that having 3 arguments is necessary here.